test_integration_app/events_api/test_events_api.py

```diff
@@ -0,0 +1,1408 @@
+        "$country"      : "US",
+        "$zipcode"      : "03257"
+      },
+      "$locations"        : [
+        {
+          "$city"         : "Seattle",
+          "$region"       : "Washington",
+          "$country"      : "US",
+          "$zipcode"      : "98112"
+        }
+      ],
+      "$categories"       : ["Personal"],
+      "$images"           : [
+        {
+          "$md5_hash"     : "0cc175b9c0f1b6a831c399e269772661",
+          "$link"         : "https://www.domain.com/file.png",
+          "$description"  : "View from the window!"
+        }
+      ],
+      "$expiration_time"  : 1549063157000 # UNIX timestamp in milliseconds
+    },
+
+    # Send this information from a BROWSER client.
+    "$browser"      : {
+      "$user_agent" :  "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36",
+      "$accept_language"  : "en-US",
+      "$content_language" : "en-GB"
+    }
+  }
+  response = client.track("$create_content", post_properties)
+  assert(response.is_ok())
+  assert response.api_status == 0, "api_status should be 0"
+  assert response.api_error_message == "OK", "api_error_message should be OK"
+
+def create_content_profile():
+  # Sample $create_content event for reviews
+  profile_properties = {
+    # Required fields
+    "$user_id"              : "fyw3989sjpqr71",
+    "$content_id"           : "profile-23412",
+
+    # Recommended fields
+    "$session_id"           : "a234ksjfgn435sfg",
+    "$status"               : "$active",
+    "$ip"                   : "255.255.255.0",
+
+    # Required $profile object
+    "$profile"              : {
+      "$body"             : "Hi! My name is Alex and I just moved to New London!",
+      "$contact_email"    : "alex_301@domain.com",
+      "$contact_address"  : {
+          "$name"         : "Alex Smith",
+          "$phone"        : "1-415-555-6041",
+          "$city"         : "New London",
+          "$region"       : "New Hampshire",
+          "$country"      : "US",
+          "$zipcode"      : "03257"
+      },
+      "$images"           : [
+        {
+          "$md5_hash"     : "0cc175b9c0f1b6a831c399e269772661",
+          "$link"         : "https://www.domain.com/file.png",
+          "$description"  : "Alex's picture"
+        }
+      ],
+      "$categories"       : [
+        "Friends",
+        "Long-term dating"
+      ]
+    },
+
+    # Send this information from a BROWSER client.
+    "$browser"      : {
+      "$user_agent" :  "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36",
+      "$accept_language"  : "en-US",
+      "$content_language" : "en-GB"
+    }
+  }
+
+  response = client.track("$create_content", profile_properties)
+  assert(response.is_ok())
+  assert response.api_status == 0, "api_status should be 0"
+  assert response.api_error_message == "OK", "api_error_message should be OK"
```

Comment:
this is the same code used in different tests. We may use something like `utils.py` file with the functions that can be reused